package.json

```diff
@@ -15,13 +15,13 @@
     "dev": "npm run build:widget && npm run dev:widget | npm run dev:app",
     "dev:app": "vite dev",
     "dev:style": "postcss --watch src/lib/widget/main.css --output package/widget.css",
-    "dev:widget": "rollup -cw",
+    "dev:widget": "tsup --watch",
     "build": "npm run build:widget && npm run build:app",
     "build:app": "vite build",
     "build:storybook": "build-storybook",
     "build:widget": "rollup -c",
```

Comment:
This was the intended `build` command to build the widget, rather than `package`. I feel like keeping this syntax helps keep the overall project stay consistent. Is there a strong reason why you want to shift to the `package` command?

Reply:
I switched it so we had testing/ development against the build system that was being used to package. I can switch back if its preferred otherwise. wanted testing and stuff to go off the "package" version

Reply:
> Can we get an example output of this PR for the package? Just a screenshot of the files and top-level dirs in the `package/` directory?

<img width="1438" alt="Screen Shot 2022-12-15 at 1 12 19 PM" src="https://user-images.githubusercontent.com/18267769/207957572-481463d1-6027-4749-8c24-c260b0abb3f1.png">